README.md

```diff
@@ -1,20 +1,52 @@
 3) Edit the global attributes in `figure_generation/graph_creation.py` to suit your tastes. (The global attributes should all be documented at the top of `figure_generation/graph_creation.py`.)
 4) Execute `python3 ./figure_generation/graph_creation.py`.
 
 Note that we did a significant amount of aesthetic primping in Illustrator to the figures generated from the included default data before we put them in the paper, but the data is all the same.
 
-## Documentation
+## Regenerating DeepCell Kiosk Paper Benchmarking Data
+
+This repository can produce the benchmarking figures from the paper, but it expects a full battery of benchmarking output files from all run conditions. It will not produce any figures as-is without output from all runs. 
+ 
+To generate a full battery of benchmarking run data (i.e., varying image numbers and GPU numbers), please complete the following benchmarking runs:
+ 
+     - 3 runs with 1 GPU and 10,000 images
+     - 3 runs with 4 GPUs and 10,000 images
+     - 3 runs with 8 GPUs and 10,000 images
+     - 3 runs with 1 GPU and 100,000 images
+     - 3 runs with 4 GPUs and 100,000 images
+     - 3 runs with 8 GPUs and 100,000 images
+     - 1 run with 1 GPU and 1,000,000 images
+     - 1 run with 4 GPUs and 1,000,000 images
+     - 1 run with 8 GPUs and 1,000,000 images
+ 
+Then, follow the instructions in [Developer docs of the kiosk-console repo](https://deepcell-kiosk.readthedocs.io/en/master/DEVELOPER.html), in addition to the constraints outlined below in the `Benchmarking Figure Recreation Settings` section, to recreate the DeepCell Kiosk paper's benchmarking figures.
+ 
+### Benchmarking Figure Recreation Settings
+ 
+When recreating the figures from the DeepCell Kiosk paper, please observe the following configuration guidelines:
+ 
+    - In the DeepCell Kiosk paper, benchmarking data was presented for clusters with maxima of 1, 4, and 8 GPUs. Choose the appropriate maximum during DeepCell Kiosk configuration for the benchmarking dataset you would like to recreate.
+    - Keep the default values for `MODEL` and `FILE` in the benchmarking YAML file.
+    - Copy the `models/NuclearSegmentation` folder in the Van Valen Lab's [kiosk-benchmarking bucket](https://console.cloud.google.com/storage/browser/kiosk-benchmarking) to the same location in your benchmarking bucket.
+    - Copy the Van Valen Lab's [zip100.zip](https://console.cloud.google.com/storage/browser/_details/kiosk-benchmarking/sample-data/zip100.zip) into the `uploads` folder of your benchmarking bucket. This file consists of 100 microscopy images, which we used as the basis for all the benchmarking runs in the DeepCell Kiosk paper.
+    - Benchmarking data was presented in the DeepCell Kiosk paper for 10,000-image, 100,000-image, and 1,000,000-image runs. Since `zip100.zip` contains 100 images, set the `COUNT` variable in the benchmarking YAML file to either 100, 1,000, or 10,000.
+
+## Miscellanea
```

Comment:
No reason to include this if it's not used.

Reply:
That was actually added to act as an umbrella for the section that follows.